dashboards/google-cloudsql/metadata.yaml

```diff
@@ -2,7 +2,7 @@ sample_dashboards:
   -
     category: Cloud SQL
     id: cloudsql-general
-    display_name: Cloud SQL General
+    display_name: CloudSQL - General
```

Comment:
Can you update these names in the readme files too? We can attempt to keep them in sync for now.

Reply:
Done.